### PR TITLE
JR2-1163 Fix pagination

### PR DESCRIPTION
**Pull request by varusakov**

### CODE REVIEW

File: src/main/webapp/app/shared/job-advertisement/job-advertisement.service.ts [hunk 1]
```diff
@@ -69,7 +69,7 @@ export class JobAdvertisementService {
             params = params.set('token', jobAdCancelRequest.token);
         }
         return this.http.patch(`${this.resourceUrl}/${jobAdCancelRequest.id}/cancel`,
-            { code }, { params: params, observe: 'response' })
+            { code }, { params, observe: 'response' })
```
Comment (pavax) [inline src/main/webapp/app/shared/job-advertisement/job-advertisement.service.ts]: why did you change this line

Comment (varusakov) [reply src/main/webapp/app/shared/job-advertisement/job-advertisement.service.ts]: @pavax to remove warning in logs during compilation
